Chinese/DesertHell/OldStory/Story.md

```diff
@@ -6,3 +6,5 @@
 有一隻白猫,
 他是一隻活过一万次的猫,
 [他讲的什么？](./Story.md)
+
+[突然](./Cat2Dog/变成够了.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.